crypto/nextgen_crypto/src/bls12381.rs

```diff
@@ -43,56 +43,44 @@ use pairing::{
 use rand::Rng;
 use serde::{Deserialize, Serialize};
 use std::ops::Deref;
-use threshold_crypto;
 
-// type alias for this unwieldy type
+/// The length of the Ed25519PrivateKey.
+pub const BLS12381_PRIVATE_KEY_LENGTH: usize = 32;
```

Comment:
Isn't it this: https://github.com/poanetwork/threshold_crypto/blob/485333db6e2611d26a2a921a6ee1e1dcc3ce4624/src/secret.rs#L11
given :
https://github.com/poanetwork/threshold_crypto/blob/master/src/lib.rs#L336
?

Reply:
True, but I as I don't have access on it `pub(crate)`, I fix it to 32, as in BLS12_381 `r` is a 255 bits, see [BLS12-381 specs](https://electriccoin.co/blog/new-snark-curve/).